examples/sample-app/README.md

```diff
@@ -118,7 +119,7 @@ This section covers how to perform all the steps of building, deploying, and upd
 
        **VAGRANT USERS**: Instead of the above command, use
 
-        $ sudo /data/src/github.com/openshift/origin/_output/local/bin/linux/amd64/openshift start --public-master=localhost --volume-dir=</absolute/path> &> openshift.log &
+        $ sudo `which openshift` start --public-master=localhost --volume-dir=</absolute/path> &> openshift.log &
```

Comment:
i don't see the point of this.  if openshift is in the path, then you can just run it.  and if it's not, which won't find it.